docs/server/source/appendices/cryptography.md

```diff
@@ -15,8 +15,9 @@ algorithm provided by the
 which is a wrapper around the optimized reference implementation
 from [http://keccak.noekeon.org](http://keccak.noekeon.org).
 
-Here's the relevant code from `bigchaindb/bigchaindb/common/crypto.py`
-(as of 11 December 2016):
+**Important**: Since selecting the Keccak hashing algorithm for SHA-3 in 2012, NIST [released a new version](https://en.wikipedia.org/wiki/SHA-3#cite_note-14) of the hash using the same algorithm but slightly different parameters. As of version 0.9, BigchainDB is using the latest version. See below for an example output of the hash function.
```

Comment:
Maybe say which version of the `pysha3` Python package we are using as of BigchainDB Server version 0.9, i.e. `pysha3==1.0b1`